docs/changelog/1_0_a6.rst

```diff
@@ -0,0 +1,63 @@
+===========
+1.0 Alpha 6
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 6 "Wolf".
+
+
+EdgeQL
+======
+
+* Introduce new enum definition syntax (:eql:gh:`#1843`).
+
+  Instead of using strings, the enums are now defined using identifiers:
+
+  .. code-block:: edgeql
+
+    CREATE SCALAR TYPE schema::Cardinality
+        EXTENDING enum<ONE, MANY>;
```

Comment:
Example should show CamelCase option, since this is our new convention.

Reply:
I changed the example.